src/overlays/actors/ovl_Bg_Dblue_Waterfall/z_bg_dblue_waterfall.c

```diff
@@ -33,69 +37,596 @@ const ActorInit Bg_Dblue_Waterfall_InitVars = {
     (ActorFunc)BgDblueWaterfall_Draw,
 };
 
-// static ColliderCylinderInit sCylinderInit = {
-static ColliderCylinderInit D_80B85370 = {
-    { COLTYPE_NONE, AT_NONE, AC_ON | AC_TYPE_PLAYER, OC1_ON | OC1_TYPE_ALL, OC2_TYPE_2, COLSHAPE_CYLINDER, },
-    { ELEMTYPE_UNK4, { 0x00000000, 0x00, 0x00 }, { 0x00CBFBB0, 0x00, 0x00 }, TOUCH_NONE | TOUCH_SFX_NORMAL, BUMP_ON, OCELEM_ON, },
+static ColliderCylinderInit sCylinderInit = {
+    {
+        COLTYPE_NONE,
+        AT_NONE,
+        AC_ON | AC_TYPE_PLAYER,
+        OC1_ON | OC1_TYPE_ALL,
+        OC2_TYPE_2,
+        COLSHAPE_CYLINDER,
+    },
+    {
+        ELEMTYPE_UNK4,
+        { 0x00000000, 0x00, 0x00 },
+        { 0x00CBFBB0, 0x00, 0x00 },
+        TOUCH_NONE | TOUCH_SFX_NORMAL,
+        BUMP_ON,
+        OCELEM_ON,
+    },
     { 50, 740, -740, { 0, 0, 0 } },
 };
 
-// static InitChainEntry sInitChain[] = {
-static InitChainEntry D_80B853A8[] = {
+Color_RGBA8 D_80B8539C = { 200, 255, 255, 255 };
+Color_RGBA8 D_80B853A0 = { 200, 200, 200, 0 };
+
+s32 func_80B83C80(Vec3f* arg0, Vec3f* arg1) {
+    f32 temp_f0 = Math3D_LengthSquared(arg0);
+    f32 temp_f2;
+
+    if (temp_f0 < 0.0000010000001f) {
```

Comment:
Is this exact float required to match? Can't you use something simpler?
```suggestion
    if (temp_f0 < 0.000001f) {
```

Reply:
Doesn't match, I tried.